projects/ngx-charts-on-fhir/src/lib/data-layer-list/annotation-list/annotation-list.component.spec.ts

```diff
@@ -26,37 +26,40 @@ describe('DatasetAnnotationListComponent', () => {
     expect(component).toBeTruthy();
   });
   describe('onCheckboxChange', () => {
-    it('should update view, control and emit the option if an option is clicked', () => {
-      spyOn(component.annotationsChange, 'emit');
-      const annotation = { label: { display: true } };
-      component.annotations = [annotation];
-      let event: any = { checked: true };
-      component.onCheckboxChange(annotation, event);
-      expect(component.annotationsChange.emit).toHaveBeenCalled();
+    it('should set display true when checked', async () => {
+      let emitted: any;
+      component.annotationsChange.subscribe((e) => (emitted = e));
+      const annotations = [{ label: { content: 'Test', display: true }, display: false }];
+      component.annotations = annotations;
+      let checkedEvent: any = [{ label: { content: 'Test', display: true }, display: true }];
```

Comment:
this is a misleading variable name. It is not a checkbox 'checked' event, it's the expected value of the annotation.